src/debug/nodejsDebugProvider.ts

```diff
@@ -103,4 +118,15 @@ export class NodejsDebugProvider implements IDebugProvider {
     public isPortRequired(): boolean {
         return true;
     }
+
+    public async getDebugArgs(): Promise<string | undefined> {
+        const debugCommand = await vscode.window.showInputBox({
+            prompt: 'Debug command for your container:',
+            placeHolder: 'Example: node --inspect app.js'
+        });
+        if (!debugCommand) {
+            return undefined;
```

Comment:
Feels like there might be a quirk here: if the user cancels out of the input box (by hitting Esc), it looks like the deployment operation will go ahead but with no additional arguments.  This may not be what the user expects from cancellation - they may expect to cancel the entire operation, not just the command gathering part.

Reply:
Good catch :+1: Not sure about which naming to use here. I used { succeeded, command }, even if succeeded doesn't satisfy me completely. If you have a better idea, let me know

Reply:
In some of my projects I have a `Cancellable<T>` type - e.g. https://github.com/deislabs/porter-vscode/blob/master/src/utils/cancellable.ts.  Would that be appropriate here perhaps?